internal/resolver/unix/unix.go

```diff
@@ -26,20 +26,28 @@ import (
 	"google.golang.org/grpc/resolver"
 )
 
-const scheme = "unix"
+const unixScheme = "unix"
+const unixAbstractScheme = "unix-abstract"
 
-type builder struct{}
+type builder struct {
+	scheme string
```

Comment:
Can we use `target.Scheme` to determine whether to prepend the null byte instead of needing this?

Reply:
It seems that we have to store the schema name inside the builder so `builder.Scheme()` can return the correct one.

Reply:
Oh yes, I missed that, sorry.